apps/api/src/bid/services/bid.service.ts

```diff
@@ -0,0 +1,15 @@
+import axios from "axios";
+import { singleton } from "tsyringe";
+
+import { RestAkashBidListResponseType } from "@src/types/rest";
+import { apiNodeUrl } from "@src/utils/constants";
+import { ListBidsResponse } from "../http-schemas/bid.schema";
+
+@singleton()
+export class BidService {
+  public async list(owner: string, dseq: string): Promise<ListBidsResponse['data']> {
+    const response = await axios.get<RestAkashBidListResponseType>(`${apiNodeUrl}/akash/market/v1beta4/bids/list?filters.owner=${owner}&filters.dseq=${dseq}`);
```

Comment:
Let's extract this in the [http package](https://github.com/akash-network/console/tree/f1a0c32422b3716010f936aa7d0a4abe38ee4762/packages/http-sdk)